### DISCUSSION TITLE
The services which are available in HttpApi.Client project

### DISCUSSION THREAD

**Discussion by liorbalmas** [abpframework/abp#7600] [Q&A]
Hi
The following page mentions the HttpApi.Client. 
https://docs.abp.io/en/abp/latest/API/Dynamic-CSharp-API-Clients#service-interface

Except from Application layer module, it seems to contain useful services such as Tenancy, Account, Permission management etc.
Where can these services be explored conveniently?

Thank you

**Comment by gterdem** [comment]
I think it is crucial to understand the module architecture as shown [here](https://docs.abp.io/en/abp/latest/Best-Practices/Module-Architecture) since all the modules are developed by this architecture.

You can check [AbpAccountHttpApiClientModule](https://github.com/abpframework/abp/blob/dev/modules/account/src/Volo.Abp.Account.HttpApi.Client/Volo/Abp/Account/AbpAccountHttpApiClientModule.cs) or [other modules](https://github.com/abpframework/abp/tree/dev/modules) if you are looking for exploring the modules.
